matplotlibrc.template

```diff
@@ -105,7 +105,7 @@ backend      : $TEMPLATE_BACKEND
 #patch.linewidth        : None     # edge width in points.
                                    # If None, use axes.linewidth when patch
                                    # is not filled.
-#patch.facecolor        : #1f77b4
+#patch.facecolor        : 1f77b4
```

Comment:
Should this one be quoted, like `grid.color`, or should the other one be unquoted?